### DISCUSSION TITLE
Focus Mode

### DISCUSSION THREAD

**Discussion by Vertikar** [aaronwmorris/indi-allsky#544] [Q&A]
Hi All, 
Trying to get my focus dialled in using focus mode, and it seems the best number is somewhere around 220-230 for me.
Does that sound about right? What is everyone else on when they've got it all nicely focused?

**Comment by aaronwmorris** [comment]
I completely missed this topic.  The focus number if highly subjective.  I am not even sure the number is actually valuable in achieving critical focus.

I generally use a video streaming mode to get the best focus.
